servicecontrol/capacity-and-planning.md

```diff
@@ -61,4 +61,4 @@ The transports supported by ServiceControl out-of-the-box (i.e. MSMQ, RabbitMQ,
 
 Azure Queues and Service Bus throughput varies significantly based on deployment options and multiple related variables inherent to cloud deployment scenarios.
 
-It is recommended that you plan and perform realistic throughput tests on ServiceControl using the transport of your choice and deployment options that are as close as possible to your planned production deployment. For additional questions or information please [contact Particular Software](http://particular.net/contactus).
\ No newline at end of file
+We recommended that customers plan and perform realistic throughput tests on ServiceControl using the transport of your choice and deployment options that are as close as possible to your planned production deployment. For additional questions or information please [contact Particular Software](http://particular.net/contactus).
```

Comment:
It is recommended that, prior to a production deployment of ServiceControl, realistic throughput tests are performed using the production transport and deployment options.